src/components/views/dialogs/RoomSettingsDialog.tsx

```diff
@@ -40,14 +40,16 @@ import { NonEmptyArray } from "../../../@types/common";
 import { PollHistoryTab } from "../settings/tabs/room/PollHistoryTab";
 import ErrorBoundary from "../elements/ErrorBoundary";
 
-export const ROOM_GENERAL_TAB = "ROOM_GENERAL_TAB";
-export const ROOM_VOIP_TAB = "ROOM_VOIP_TAB";
-export const ROOM_SECURITY_TAB = "ROOM_SECURITY_TAB";
-export const ROOM_ROLES_TAB = "ROOM_ROLES_TAB";
-export const ROOM_NOTIFICATIONS_TAB = "ROOM_NOTIFICATIONS_TAB";
-export const ROOM_BRIDGES_TAB = "ROOM_BRIDGES_TAB";
-export const ROOM_ADVANCED_TAB = "ROOM_ADVANCED_TAB";
-export const ROOM_POLL_HISTORY_TAB = "ROOM_POLL_HISTORY_TAB";
+export enum RoomSettingsTab {
```

Comment:
I thought that we decided to use union instead of enum

Reply:
Why not const enums then?

Reply:
Yes, it should work too. https://www.typescriptlang.org/docs/handbook/enums.html#const-enums I'm not very enthusiast after reading the pitfalls but yeah that's better than traditional enum.